CMakeLists.txt

```diff
@@ -5,8 +5,35 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   add_compile_options(-Wall -Wextra -Wpedantic)
 endif()
 
+# Dependencies
+
+find_package(rclcpp REQUIRED)
+find_package(sensor_msgs REQUIRED)
 find_package(ament_cmake REQUIRED)
 
+# Targets
+
+include_directories(include)
```

Comment:
Don't know if there is a difference (for these drivers) between "include_directories()" and "target_include_directories()" but may need to be changed

Reply:
`target_include_directories` just includes them for a specific target; here I've included them for all targets.